### PR TITLE
Type aliases and maps

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

File: concepts/maps/about.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+let map2 = map.from_list([
+  #("name", "Gleam"),
+  #("colour", "Pink"),
+])
+
+// Add a value to a map
+let map3 = map.insert(map2, "website", "https://gleam.run")
+
+// Get a value from a map
+let name = map.get(map3, "name")
```
Comment (bcpeinhardt) [inline concepts/maps/about.md:25]: Maybe use an assert to show the return type is result?

Comment (lpil) [reply concepts/maps/about.md:25]: Assert hasn't been taught yet, I'll add a comment

File: concepts/maps/about.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Maps in Gleam are the data structure for storing information in key-value pairs. In other languages, these might also be known as associative arrays, hashes, or dictionaries.
+
+Any type can be used for the keys and values in a map, and they do not guarantee the order of their entries when accessed or returned.
+
+## Working with maps
+
+Maps are created and manipulated using functions from the `gleam/map` module.
+
+```gleam
+// Create an empty map
```
[the remark this answers is not in the document]
Comment (lpil) [reply concepts/maps/about.md:12]: This wouldn't be valid Gleam any more so I don't think we should show it as an example.

[the remark this answers is not in the document]
Comment (giacomocavalieri) [reply concepts/maps/about.md:12]: Ooh right I didn't think about that!

**Pull request merged**